draft-ietf-core-coap-tcp-tls.md

```diff
@@ -1652,7 +1653,16 @@ Subprotocol Definition.
 :   [RFCthis]
 {: vspace='0'}
 
+## CoAP Option Numbers Registry
 
+IANA is requested to add [RFCthis] to the references for the following entries in the "CoAP Option Numbers"
+registry defined by {{RFC7252}} and updated by {{RFC7959}}:

```

Comment:
This wording conflates the registry (which was defined by RFC 7252) with the entries (which were registered in RFC 7959).   How about:

```
IANA is requested to add [RFCthis] to the references for the following entries registered by {{RFC7959}} in the "CoAP Option Numbers" registry defined by {{RFC7252}}:
```